src/systems/hydrodynamics/Hydrodynamics.hh

```diff
@@ -72,6 +72,8 @@ namespace systems
   ///     separately. If no namespace is given then the plugin listens on
   ///     the `/ocean_current` topic for a `Vector3d` message. Otherwise it
   ///     listens on `/model/{namespace name}/ocean_current`.[String, Optional]
+  ///   <defaultCurrent> - A generic current.
```

Comment:
Nit: Typically, the SDF parameters are `snake_case`. I recommend to change it to `default_current`.

Reply:
Should I also change `<waterDensity>` to `<water_density>`,

Reply:
You can but leave the original as well (deprecated) and we'll remove in the next version.

Reply:
All changes look good to me but my only suggestion is to show a warning or similar if the user uses `waterDensity`. We'll remove support for that parameter in the next release ("G").